worlds/banjo_tooie/BTClient.py

```diff
@@ -0,0 +1,1248 @@
+                if not program or program.poll() != None:
+                    asyncio.create_task(patch_and_run(False))
+            else:
+                logger.error("No file selected...")
+                return False
+        else:
+            bt_options.program_path = ""
+            bt_options._changed = True
+            logger.info("Autostart disabled.")
+        return True
+
+    def _cmd_rom_path(self, path=""):
+        """Sets (or unsets) the file path of the vanilla ROM used for patching."""
+        bt_options.rom_path = path
+        bt_options._changed = True
+        if path:
+            logger.info("rom_path set!")
+        else:
+            logger.info("rom_path unset!")
+        return True
+
+    def _cmd_patch_path(self, path=""):
+        """Sets (or unsets) the folder path of where to save the patched ROM."""
+        bt_options.patch_path = path
+        bt_options._changed = True
+        if path:
+            logger.info("patch_path set!")
+        else:
+            logger.info("patch_path unset!")
+        return True
+
+    def _cmd_program_args(self, path=""):
+        """Sets (or unsets) the arguments to pass to the automatically run program. Defaults to passing the lua to Bizhawk."""
+        bt_options.program_args = path
+        bt_options._changed = True
+        if path:
+            logger.info("program_args set!")
+        else:
+            logger.info("program_args unset!")
+        return True
+
+    def _cmd_n64(self):
+        """Check N64 Connection State"""
+        if isinstance(self.ctx, BanjoTooieContext):
+            logger.info(f"N64 Status: {self.ctx.n64_status}")
+
+    def _cmd_deathlink(self):
+        """Toggle deathlink from client. Overrides default setting."""
+        if isinstance(self.ctx, BanjoTooieContext):
+            self.ctx.deathlink_client_override = True
+            self.ctx.deathlink_enabled = not self.ctx.deathlink_enabled
+            async_start(self.ctx.update_death_link(self.ctx.deathlink_enabled), name="Update Deathlink")
+
+    def _cmd_taglink(self):
+        """Toggle taglink from client. Overrides default setting."""
+        if isinstance(self.ctx, BanjoTooieContext):
+            self.ctx.taglink_client_override = True
+            self.ctx.taglink_enabled = not self.ctx.taglink_enabled
+            async_start(self.ctx.update_tag_link(self.ctx.taglink_enabled), name="Update Taglink")
+
+    # def _cmd_tag(self):
+    #     """Toggle a tag for Taglink."""
+    #     if isinstance(self.ctx, BanjoTooieContext):
+    #         async_start(self.ctx.send_tag_link(), name="Send Taglink")
+
+
+class BanjoTooieContext(CommonContext):
+    command_processor = BanjoTooieCommandProcessor
+    items_handling = 0b111 #full
+
+    def __init__(self, server_address, password):
+        super().__init__(server_address, password)
+        self.game = "Banjo-Tooie"
+        self.n64_streams: (StreamReader, StreamWriter) = None # type: ignore
```

Comment:
```suggestion
        self.n64_streams: tuple[StreamReader, StreamWriter] | None = None
```
I guess you're aware by the type comment, but this annotation (although it's also in other worlds) is invalid. This is how you should write it. Also you should import `StreamReader` and `StreamWriter` for this.